jetty-ee10/jetty-ee10-servlet/src/main/java/org/eclipse/jetty/ee10/servlet/Dispatcher.java

```diff
@@ -645,26 +655,13 @@ public String getQueryString()
         @Override
         public String getRequestURI()
         {
-            return _uri == null ? null : _uri.getPath();
-        }
-
-        @Override
-        public Object getAttribute(String name)
-        {
-            switch (name)
-            {
-                // TODO
-                default:
-                    return super.getAttribute(name);
-            }
+            return _uri == null ? super.getRequestURI() : _uri.getPath();
```

Comment:
Is it possible to have an error page dispatch with a null URI?

Reply:
in diagnosing the behavior on null `_servletPathMapping` it was discovered that having `_uri` be null is also no possible.

So I changed all of these to `Objects.requireNonNull(_uri)` too